.github/workflows/dash.yaml

```diff
@@ -6,6 +6,10 @@ on:
     branches:
       - main
 
+concurrency:
```

Comment:
We have concurrency for most others to interrupt if a new instance of the workflow is started, do not think it hurts here either